test/e2e/setup_testcontainers/containers.py

```diff
@@ -71,22 +71,15 @@ def _get_fuseki(
         .with_env("ADMIN_PASSWORD", "test")
     )
     fuseki.start()
-    wait_for_logs(fuseki, f"Server .+ Started .+ on port {FUSEKI_INTERNAL_PORT}")
+    wait_for_logs(fuseki, rf"Start Fuseki \(http={FUSEKI_INTERNAL_PORT}\)")
```

Comment:
The log message has changed. So, waiting for the old log message would take forever.